README.md

```diff
@@ -104,12 +95,17 @@ The API methods are _mostly_ just wrappers around [`signature_pad`'s API](https:
 
 ## Example
 
+Have a look at the [`example/`](example/) directory.
+
+This example can be run with:
+
 ```bash
-$ npm start
+npm start
 ```
```

Comment:
unsure if this should be on the same line instead of a full code block. would give running it a little bit less attention (as intended) but also obscure an instruction (not intended). 🤷‍♂